### DISCUSSION TITLE
Is it possible to dynamically whitelist subdomains?

### DISCUSSION THREAD

**Discussion by MehdiRaash** [Web3Auth/Web3Auth#1239] [Help]
In my case, subdomains are being created dynamically, so I need to whitelist them all on the fly.

**Comment by MehdiRaash** [comment]
seems cannot remove my question ;-) , @yashovardhan please make it duplicate.

**Comment by himanshuchawla009** [comment]
refer to this discussion: https://github.com/orgs/Web3Auth/discussions/351
